twake/frontend/src/app/components/user-card/user-card.scss

```diff
@@ -1,3 +1,12 @@
+.custom-message {
+  width: calc(100% + 16px);
+  margin-left: -8px;
+  padding: 5px 0;
+  margin-bottom: 6px;
+  color: white;
+  font-size: 11px;
+}
+
```

Comment:
Also, we don't use css anymore for all new component, instead we use tailwind css:
https://tailwindcss.com/docs/font-size

So I will remove this change and replace it by (according to the doc)
`text-white bg-zinc-700 flex items-center content-center text-center text-xs p-2 -mt-4 -mx-4 mb-2`

Reply:
It seems you already did almost all of it using tailwind but then you mixed it with classical css, you were almost there :)